Reneiski_Homework1/Zavod/Worker.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using System.Reflection.Metadata.Ecma335;
+
+namespace WorkerNamespace
+{
+    public class Worker
```

Comment:
Please add method that shows info about first name, last name, age, position of worker

Reply:
Fixed